client/src/pages/recording-view/hooks/use-spectrogram-context.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMeta } from '@/api/metadata/queries';
+import { SigMFMetadata } from '@/utils/sigmfMetadata';
+import React, { createContext, useContext, useState } from 'react';
+
+interface SpectrogramContextProperties {
+  type: string;
+  account: string;
+  container: string;
+  filePath: string;
+  magnitudeMin: number;
+  setMagnitudeMin: (magnitudeMin: number) => void;
+  magnitudeMax: number;
+  setMagnitudeMax: (magnitudeMax: number) => void;
+  colmap: string;
+  setColmap: (colmap: string) => void;
+  windowFunction: string;
+  setWindowFunction: (windowFunction: string) => void;
+  fftSize: number;
+  setFFTSize: (fftSize: number) => void;
+  spectrogramHeight: number;
+  setSpectrogramHeight: (spectrogramHeight: number) => void;
+  spectrogramWidth: number;
+  setSpectrogramWidth: (spectrogramWidth: number) => void;
+  fftStepSize: number;
+  setFFTStepSize: (fftStepSize: number) => void;
+  includeRfFreq: boolean;
+  setIncludeRfFreq: (includeRfFreq: boolean) => void;
+  meta: SigMFMetadata;
+}
+
+export const SpectrogramContext = createContext<SpectrogramContextProperties>(null);
+
+export function SpectrogramContextProvider({
+  children,
+  type,
+  account,
+  container,
+  filePath,
+  seedValues = {
+    magnitudeMin: -100,
```

Comment:
Is there any doc anywhere about why these are the seed values?

Reply:
I am jsut getting the current ones in the current docs.